network/endpoint.go

```diff
@@ -93,6 +93,7 @@ type EndpointInfo struct {
 	SkipDefaultRoutes        bool
 	HNSEndpointID            string
 	HostIfName               string
+	Endpoint                 endpoint
```

Comment:
its not required to add whole endpoint structure.. it will be duplicating fields here